src/main/java/de/chojo/repbot/commands/repsettings/handler/EmojiInfo.java

```diff
@@ -0,0 +1,60 @@
+package de.chojo.repbot.commands.repsettings.handler;
+
+import de.chojo.jdautil.interactions.slash.structure.handler.SlashHandler;
+import de.chojo.jdautil.wrapper.EventContext;
+import de.chojo.repbot.dao.provider.Guilds;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import org.jetbrains.annotations.PropertyKey;
+
+import java.util.List;
+
+import static de.chojo.repbot.util.EmojiDebug.DONOR_LIMIT;
+import static de.chojo.repbot.util.EmojiDebug.DONOR_NOT_IN_CONTEXT;
+import static de.chojo.repbot.util.EmojiDebug.EMPTY_CONTEXT;
+import static de.chojo.repbot.util.EmojiDebug.FOUND_THANKWORD;
+import static de.chojo.repbot.util.EmojiDebug.ONLY_COOLDOWN;
+import static de.chojo.repbot.util.EmojiDebug.PROMPTED;
+import static de.chojo.repbot.util.EmojiDebug.RECEIVER_LIMIT;
+import static de.chojo.repbot.util.EmojiDebug.TARGET_NOT_IN_CONTEXT;
+import static de.chojo.repbot.util.EmojiDebug.TOO_OLD;
+
+public class EmojiInfo implements SlashHandler {
+
+    private final Guilds guilds;
+
+    public EmojiInfo(Guilds guilds) {
+        this.guilds = guilds;
+    }
+
+    @Override
+    public void onSlashCommand(SlashCommandInteractionEvent event, EventContext context) {
+        var generalSettings = guilds.guild(event.getGuild()).settings().general();
+        event.reply(getBooleanMessage(context, generalSettings.isEmojiDebug(),
+                "command.repsettings.emojidebug.message.true", "command.repsettings.emojidebug.message.false")
+                    + "\n" + context.localize(emojiExplanation())).queue();
+    }
+
+    private String emojiExplanation() {
+        var emojis = List.of(
+                String.format("$%s$", "command.repsettings.emojidebug.message.title"),
+                emojiString(FOUND_THANKWORD, "command.repsettings.emojidebug.message.found"),
+                emojiString(ONLY_COOLDOWN, "command.repsettings.emojidebug.message.cooldown"),
+                emojiString(EMPTY_CONTEXT, "command.repsettings.emojidebug.message.noReceiver"),
+                emojiString(TARGET_NOT_IN_CONTEXT, "command.repsettings.emojidebug.message.noRecentMessages"),
+                emojiString(DONOR_NOT_IN_CONTEXT, "command.repsettings.emojidebug.message.noDonor"),
+                emojiString(TOO_OLD, "command.repsettings.emojidebug.message.tooold"),
+                emojiString(PROMPTED, "command.repsettings.emojidebug.message.prompted"),
+                emojiString(DONOR_LIMIT, "command.repsettings.emojidebug.message.donorLimit"),
+                emojiString(RECEIVER_LIMIT, "command.repsettings.emojidebug.message.receiverLimit")
+        );
+        return String.join("\n", emojis);
+    }
+
+    private String getBooleanMessage(EventContext context, boolean value, String whenTrue, String whenFalse) {
+        return context.localize(value ? whenTrue : whenFalse);
+    }
```

Comment:
In the abuse protection command, this has been extracted to a utility interface. However, Abuse Protection migration is not yet merged so this is currently fine

Reply:
Aye we will need to do some kind of general refactoring in the end. there is some stuff we can improve after merging all commands.